app/views/kaui/transactions/_control_plugin_names.html.erb

```diff
@@ -8,7 +8,7 @@
     </tr>
     </thead>
     <tbody>
-    <% (@control_plugin_names || ['']).each_with_index do |control_plugin_name, index| %>
+    <% (@control_plugin_names ||= ['']).each_with_index do |control_plugin_name, index| %>
```

Comment:
Let's keep it that way for now (it was already clunky anyways) but in the future, it would be better to push that logic back to the controller, so that the view only deals with the presentation layer (i.e. not too much logic).